clang/test/CXX/drs/cwg7xx.cpp

```diff
@@ -71,6 +76,17 @@ namespace cwg712 { // cwg712: partial
 #endif
 }
 
+namespace cwg713 { // cwg713: yes
```

Comment:
Please, run your test through various Clang versions before claiming availability. In this case, this is rejected by 18, but accepted on trunk: https://godbolt.org/z/dbq67deWE
So the right status might be 19 or 20 depending on when we started to accept this.